lib/styles/settings/typography/_tokens.scss

```diff
@@ -349,7 +349,7 @@
 }
 
 @mixin TextXLRegularItalic() {
-	--token: text-xl-regular-italic;
+	--token: text-xl-regularitalic;
```

Comment:
it's hard to read when it's all lowercase

Reply:
Fixed.